public/app/features/variables/inspect/utils.ts

```diff
@@ -78,6 +78,11 @@ export const getUnknownVariableStrings = (variables: VariableModel[], model: any
       continue;
     }
 
+    if (match.indexOf('${__') !== -1) {
```

Comment:
```suggestion
    if (match.includes('${__')) {
```
Nit: I think this is a bit easier to reason about.